src/app/loadout-builder/ModPicker.tsx

```diff
@@ -158,22 +173,10 @@ class ModPicker extends React.Component<Props, State> {
     const { defs, mods, language, onClose, isPhonePortrait } = this.props;
     const { query, height, lockedArmor2Mods } = this.state;
 
-    const Armor2ModPlugCategoriesTitles = {
-      [Armor2ModPlugCategories.general]: 'LB.General',
-      [Armor2ModPlugCategories.helmet]: 'LB.Helmet',
-      [Armor2ModPlugCategories.gauntlets]: 'LB.Gauntlets',
-      [Armor2ModPlugCategories.chest]: 'LB.Chest',
-      [Armor2ModPlugCategories.leg]: 'LB.Legs',
-      [Armor2ModPlugCategories.classitem]: 'LB.ClassItem',
-      ['seasonal']: 'LB.Seasonal'
-    };
-
-    const order = [...Object.values(Armor2ModPlugCategories), 'seasonal' as 'seasonal'].map(
-      (category) => ({
-        category,
-        nameTranslation: Armor2ModPlugCategoriesTitles[category]
-      })
-    );
+    const order = [...Object.values(ModPickerCategories)].map((category) => ({
```

Comment:
`Object.values` is already an array... no need to copy it into another array.

Reply:
Done